daemon/main.c

```diff
@@ -552,8 +552,25 @@ main_terminate(
 
 static void move_to_root_cgroup(void) {
   FILE *f;
+  const char *cgroup_task_fname = NULL;
 
-  f = fopen("/sys/fs/cgroup/cpu/tasks", "w");
+  // Determine if we are using cgroup v1 or v2
+  f = fopen("/sys/fs/cgroup/cpu/cpu.rt_runtime_us", "rt");
```

Comment:
A bit late now... but it is a bit strange specifying the "t" fopen() flag here when it is a Windows-specific flag.

Reply:
AIUI it's the default on linux, so yes not really needed (but doesn't cause any issues) - again this was inspired by corosync, so may just be some developer's style their I guess...